### ISSUE TITLE
[BUG] lmformatenforcer integration seems to be broken on new versions

### ISSUE THREAD

**Issue by hvico** [turboderp/exllamav2#696]
### OS

Linux

### GPU Library

CUDA 12.x

### Python version

3.10

### Pytorch version

2.5

### Describe the bug

When running lmformatenformatenforcer integration for JSON output I get:

Error during execution: 'ExLlamaV2TokenEnforcerFilter' object has no attribute 'background_drop'

and if I comment that call in the generator code it fails with another missing stuff related with logit mask.

### Reproduction steps

Try to enforce json output on v0.2.6

### Expected behavior

Integration works.

### Acknowledgements

- [X] I have looked for similar issues before submitting this one.
- [X] I understand that the developers have lives and my issue will be answered when possible.
- [X] I understand the developers of this program are human, and I will ask my questions politely.

**Comment by hvico** [comment]
Thanks for the workaround!
